linkerd/stack/src/lib.rs

```diff
@@ -61,6 +64,11 @@ pub use tower::{
     Service,
 };
 
+pub type BoxFutureService<S, E = linkerd_error::Error> = FutureService<
+    std::pin::Pin<Box<dyn std::future::Future<Output = Result<S, E>> + Send + 'static>>,
+    S,
+>;
```

Comment:
nit: did this sneak in from another PR accidentally? i don't see where it's used here...

Reply:
yeah. and we're not even using it in the other branches anymore. I'll kill it.